src/rgw/rgw_iam_policy_keywords.gperf

```diff
@@ -109,6 +109,7 @@ Null, TokenKind::cond_op, TokenID::Null, (uint64_t) Type::null, true, true
 #s3:authType, TokenKind::cond_key, TokenID::s3authType, (uint64_t) Type::string, true, false
 #s3:signatureAge, TokenKind::cond_key, TokenID::s3signatureAge, (uint64_t) Type::number, true, false
 #s3:x-amz-content-sha256, TokenKind::cond_key, TokenID::s3x_amz_content_sha256, (uint64_t) Type::string, true, false
+#s3:RequestObjectTag, TokenKind::cond_key, TokenID::s3RequestObjectTag, (uint64_t) Type::string, true, false
 #
```

Comment:
In the future the 'cond_key' class of tokens may disappear. Marcus suggested that the way STS is designed, it may be appropriate to allow any string as a condition key.

This is one of those things where I should get an S3 account and try experimenting on it to see exactly what Amazon's behavior is.

No problem with your commit, just a note.